### ISSUE TITLE
Integrator: Having trouble with increase by and decrease by searches

### ISSUE THREAD

**Issue by Dog** [openai/retro#46]
## Issue summary

I may just be misunderstanding the usage, but I'm currently trying to use the Integrator to find memory addresses. 

### Steps

1) Start the game
2) Create a search named "lives". Search for addresses that "Is"  the total lives (in my case eight)
3) Now lose one life to bring the total to seven.
4) Select the set of addresses returned by the "lives" search.
5) Create a search called "lives2". Search the set for addresses that "Decreased By" one.
6) Select the set of addresses returned by the "lives2" search.

### Expectation

We would see any address from the "lives" set which decreased from 8 to 7 in the set "lives2".

### What I See

I received an empty set. I used a known memory address as an attempt to perform a sanity check.

## System information

- Mac OS X 10.13.3
- Python 3.6.4
- develop branch (0.6.0.dev17+gba95d9b)

**Comment by christopherhesse** [comment]
I could be misinterpreting this, but I think you don't want to create a new search, just use the existing lives search and search for "decreased by" on that.

**Comment by Dog** [comment]
For the most part I've been using Rocket Knight Adventures. I don't have all my notes in front of me at the moment, but later today I can try to flesh this out further since it may legitimately be an edge case.

**Comment by Dog** [comment]
@christopherhesse This was my problem. I didn't realize I needed to use the original search by name for "Increased by" and "Decreased by". I thought by selecting "lives" in the first table I was performing the search on those results. Essentially I thought I was telling the search feature to create a new set based off that selection.

I believe part of my confusion was because I was able to deselect in the list of searches by clicking the empty space in the table after the first or second searches. However by having three or more searches at a time it becomes clear that isn't how its meant to work.

Thanks!
